src/components/DocumentList/DocumentList.stories.js

```diff
@@ -67,7 +65,7 @@ storiesOf("Document List", module)
       return (
         <DocumentList
           onSelected={container.onSelected}
-          documents={object("documents", data.oneLongName)}
+          documents={array("documents", data.oneLongName)}
```

Comment:
Knobs should be specific. Doing so makes the UI experience better, since it constrains input accordingly.

Reply:
Completed